lib/browser/electron.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Browser
+  class Electron < Base
```

Comment:
Similar code found in 7 other locations (mass = 21) ([lib/browser/electron.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/electron.rb#L4-L19), [lib/browser/alipay.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/alipay.rb#L4-L18), [lib/browser/firefox.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/firefox.rb#L4-L18), [lib/browser/micro\_messenger.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/micro_messenger.rb#L4-L18), [lib/browser/nokia.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/nokia.rb#L4-L18), [lib/browser/phantom\_js.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/phantom_js.rb#L4-L18), [lib/browser/uc\_browser.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/uc_browser.rb#L4-L18), and [lib/browser/weibo.rb#4](https://github.com/fnando/browser/blob/24d26343715008f806f9a23b7959aac9634aa3b2/lib/browser/weibo.rb#L4-L18))